client/state/reader/follows/test/actions.js

```diff
@@ -26,34 +34,88 @@ describe( 'actions', () => {
 		const actions = require( '../actions' );
 		recordFollow = actions.recordFollow;
 		recordUnfollow = actions.recordUnfollow;
+		requestFollows = actions.requestFollows;
 	} );
 
 	const spy = sinon.spy();
+	const dispatchSpy = sinon.stub();
+	dispatchSpy.withArgs( sinon.match.instanceOf( Promise ) ).returnsArg( 0 );
 
 	beforeEach( () => {
 		spy.reset();
+		dispatchSpy.reset();
 	} );
 
 	describe( '#recordFollow', () => {
 		it( 'should dispatch an action when a URL is followed', () => {
-			const dispatchSpy = sinon.stub();
-			dispatchSpy.withArgs( sinon.match.instanceOf( Promise ) ).returnsArg( 0 );
 			recordFollow( 'http://discover.wordpress.com' )( dispatchSpy );
 			expect( dispatchSpy ).to.have.been.calledWith( {
 				type: READER_FOLLOW,
-				url: 'http://discover.wordpress.com'
+				payload: { url: 'http://discover.wordpress.com' },
 			} );
 		} );
 	} );
 
 	describe( '#recordUnfollow', () => {
 		it( 'should dispatch an action when a URL is unfollowed', () => {
-			const dispatchSpy = sinon.stub();
-			dispatchSpy.withArgs( sinon.match.instanceOf( Promise ) ).returnsArg( 0 );
 			recordUnfollow( 'http://discover.wordpress.com' )( dispatchSpy );
 			expect( dispatchSpy ).to.have.been.calledWith( {
 				type: READER_UNFOLLOW,
-				url: 'http://discover.wordpress.com'
+				payload: { url: 'http://discover.wordpress.com' }
+			} );
+		} );
+	} );
+
+	describe( '#requestFollows', () => {
+		context( 'success', () => {
```

Comment:
I like that you create a `context` so that you can separate out the nock calls.

I'm stealing that :)